xml/installation-kvm_xpointer.xml

```diff
@@ -530,7 +530,7 @@ git commit -m "My config or other commit message"</screen>
   </itemizedlist>
   <para>
    If you chose this method for installing your baremetal hardware, skip
-   forward to the <xref linkend="install_kvm"/> step.
+   forward to the <!-- FIXME: this seems to be the wrong reference <xref linkend="install_kvm"/> - sknorr, 2018-01-19 --> step.
```

Comment:
There is a handful of FIXMEs in this file that need to be, er, fixed. I created a Trello card for this: https://trello.com/c/fYtlCjok